### PR TITLE
Add docs for installing pangeo_forge_recipes

### PR DESCRIPTION
**Pull request by rwegener2**
## What has been built?
A new docs page was added for installing `pangeo-forge-recipes`. It recommends `pip`.

## How was it done?
1. A new markdown file was added to `docs`
2. `index.md` was updated

## How can it be tested?
Run docs from the CI test.  Look for a new page called "Installing Pangeo Forge Recipes"

### CODE REVIEW

**Comment by rwegener2** [comment]
Update from meeting with @cisaacstern
pip is questionable, see #292 

Instead:
- to start just use binder
- for more in depth local dev use CI install

**Comment by rwegener2** [comment]
This PR has been updated after yesterday's discussion with @cisaacstern. The page now recommends:
1. Get started use Binder
2. Longer term use the CI build

I used a `git clone -b 0.8.0 https://github.com/pangeo-forge/pangeo-forge-recipes.git` for the CI install. Pro: thie version of PFR should not at any moment be accidentally broken. Con: this needs to get changed manually with each new release of PFR.

If anyone thinks I should switch to a regular `git clone` I can do that.

### Merge?
I think this PR is ready for review

**Comment by rabernat** [comment]
We will update this conda and pip install instructions once https://github.com/conda-forge/staged-recipes/pull/16146 is sorted out.

**Comment by rwegener2** [comment]
I just switched the narrative for this PR back to `pip`. I also mention the sandbox.

**Comment by rwegener2** [comment]
That's amazing!  Many 👏🏻 s to you both for getting the `conda` install setup!

I just updated the documentation:
1. conda is preferred
2. sandbox/binder is also an option

@cisaacstern @rabernat ready for re-review

File: docs/installation.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+`conda config --add channels conda-forge`
+`conda config --set channel_priority strict`
+
+If those steps are complete you can install the library with `conda install pangeo-forge-recipes`.
+
+## Remote access
+
+Pangeo Forge also has a [code sandbox](https://github.com/pangeo-forge/sandbox) which has `pangeo-forge-recipes` installed and template documents for creatig a recipe. [Click here](https://mybinder.org/v2/gh/pangeo-forge/sandbox/binder) to launch the sandbox.
+
+You'll also notice that the tutorials have binder links. The binder links can be used to explore `pangeo-forge-recipes` without setting up a local environment.
```
Comment (rabernat) [inline docs/installation.md:20]:
```suggestion
You'll also notice that the tutorials have Binder links. The binder links can be used to explore `pangeo-forge-recipes` without setting up a local environment.
```

**Review by rabernat — COMMENTED** [review]
Looking good. Two  minor nits.

**Pull request merged**